docs/UserGuide.md

```diff
@@ -292,23 +294,24 @@ Examples:
 
 #### Editing an event : `edit`
 
-Edits an existing event in the event list. DATE_TIME format follows the addEvent command.
+Edits an existing event in the event list. DATE_TIME format follows the `add -e` command.
```

Comment:
hmm, perhaps we should mention the `DATE_TIME` format at the start of the events instead of in the add command?

Reply:
I don't think it is necessary as the add command description states the details of the DATE_TIME as well as description, just like the contact side.